_schemaV2.graphql

```diff
@@ -2458,6 +2458,7 @@ type Artwork implements Node & Searchable & Sellable {
   priceCurrency: String
   priceIncludesTax: Boolean
   priceIncludesTaxDisplay: String
+  priceListed: Float
```

Comment:
I saw you initially had this as the `Money` type, which would be helpful centralizing the conversion logic in the API. Curious if there is challenge using that.

Reply:
In addition to displaying the price, we also want to use it in a calculation to display the discounted price. Does the `Money` type have a numeric field as well as a text field?

Reply:
I didn't have the conversion in mind when switching to Float, I thought the use of Money was more related to formatting and displaying the currency and since we already have a field with the display text ready, I thought that using Money here would be redundant.

The only challenge was to convert the value to cents before using the helpers and that it requires a currency for it work properly.

I haven't tried converting the values to other currencies though.

Reply:
Sorry for the confusion. By conversion, I meant major <-> minor unit conversion. The Gravity `price_listed` in major unit is not recommended, and I'd love to stop propagating that more.

> Does the Money type have a numeric field as well as a text field?

The `Money` type exposes the amount in [minor unit](https://github.com/artsy/metaphysics/blob/84314447e8401678f7a3f9a4991f59818a89990d/src/schema/v2/fields/money.ts#L162), [major unit](https://github.com/artsy/metaphysics/blob/84314447e8401678f7a3f9a4991f59818a89990d/src/schema/v2/fields/money.ts#L177), and formatted string. You can see an example query of the type in [`internationalShippingFee`](https://github.com/artsy/metaphysics/blob/84314447e8401678f7a3f9a4991f59818a89990d/src/schema/v2/artwork/__tests__/artwork.test.js#L2429C11-L2433).